Kernel/System/Ticket/Event/NotificationEvent.pm

```diff
@@ -373,39 +373,26 @@ sub _NotificationFilter {
 
     my $ArticleObject = $Kernel::OM->Get('Kernel::System::Ticket::Article');
 
+    # get not ticket related attributes
+    my $IgnoredAttributes = $Kernel::OM->Get('Kernel::Config')->Get('Frontend::Admin::AdminNotificationEvent::IgnoredAttributes');
+    my %IgnoredAttributesHash;
+    if ( IsHashRefWithData( $IgnoredAttributes ) ) {
+        for my $Key ( keys %{$IgnoredAttributes} ) {
+            if ( IsHashRefWithData( $IgnoredAttributes->{$Key} ) ) {
+                %IgnoredAttributesHash = ( %IgnoredAttributesHash, %{ $IgnoredAttributes->{$Key} } );
+            }
+        }
+    }   
```

Comment:
The code must then be adapted accordingly. (see above)

Furthermore, the SysConfig has nothing to do directly with `Frontend::Admin::AdminNotificationEvent`. The config is only used in the backend. Of course, the SysConfig is used later but then only in the backend.